RibonDAO/app
# [PROD3-123] New screen to zero tickets

# Description

- New screen to zero tickets

## Does this pull request close any open issues?

- Put here the issue that must be closed with this PR
- eg: closes [#PROD3-123]()

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Donate a ticket and click the button of 0 tickets
